### PR TITLE
Added new waypoint for North California

### PR DESCRIPTION
**Pull request by jyothipr**

### CODE REVIEW

File: src/main/java/org/jason/fgcontrol/flight/position/KnownPositions.java [hunk 1]
```diff
@@ -106,4 +106,10 @@ private KnownPositions() {}
     public final static WaypointPosition RATLAM_CITY = new WaypointPosition(23.33676, 75.00124, "Ratlam City");
     public final static WaypointPosition KOTA_CITY = new WaypointPosition(25.17353, 75.76456, "Kota City");
     public final static WaypointPosition DELHI_CITY = new WaypointPosition(28.64428, 76.92842, "Delhi City");
+
+    //North California Tour
+    public final static WaypointPosition SFO_CITY = new WaypointPosition(37.8199328, -122.4808354, "SFO City");
```
Comment (jas0ndiamond) [inline src/main/java/org/jason/fgcontrol/flight/position/KnownPositions.java]: This isn't explicitly documented but 5 decimals places of precision for lat/lon is suitable for our purposes.

File: src/main/java/org/jason/fgcontrol/flight/position/KnownRoutes.java [hunk 2]
```diff
@@ -19,6 +19,18 @@ private KnownRoutes() {}
 
     private final static Logger LOGGER = LoggerFactory.getLogger(KnownRoutes.class);
 
+    public final static ArrayList<WaypointPosition> NA_CALIFORNIA = new ArrayList<WaypointPosition>() {
+        // Starting position for the route: Stanford -> SFO -> SANJOSE -> SACRAMENTO -> TAHOE -> Stanford with starting position of Standford University location 37.4276642,-122.1726403
```
Comment (jas0ndiamond) [inline src/main/java/org/jason/fgcontrol/flight/position/KnownRoutes.java]: Add the start heading in this comment too. From Stanford to SFO it looks like the heading is 328.11.

File: src/main/java/org/jason/fgcontrol/flight/position/KnownPositions.java (hunk 1 above)
Comment (jyothipr) [reply src/main/java/org/jason/fgcontrol/flight/position/KnownPositions.java]: Updated lat/lon to 5 decimals

File: src/main/java/org/jason/fgcontrol/flight/position/KnownRoutes.java (hunk 2 above)
Comment (jyothipr) [reply src/main/java/org/jason/fgcontrol/flight/position/KnownRoutes.java]: updated comments with initial heading

**Pull request merged**